src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/utils.hpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "internal_properties.hpp"
+#include "utils/cpu_test_utils.hpp"
+#include "openvino/runtime/system_conf.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+#define SNIPPETS_TESTS_STATIC_SHAPES(...) static_shapes_to_test_representation(std::vector<std::vector<ov::Shape>>{__VA_ARGS__})
+
+static inline bool is_bf16_supported() {
+    return ov::with_cpu_x86_bfloat16() || ov::with_cpu_x86_avx512_core_amx_bf16();
+}
+
+static inline std::vector<std::vector<element::Type>> precision_f32(size_t count) {
+    std::vector<std::vector<element::Type>> prc;
+    prc.emplace_back(std::vector<element::Type>(count, element::f32));
+    return prc;
+}
+
+static inline std::vector<std::vector<element::Type>> precision_bf16(size_t count) {
+    std::vector<std::vector<element::Type>> prc;
+    if (is_bf16_supported())
+        prc.emplace_back(std::vector<element::Type>(count, element::bf16));
+    return prc;
+}
```

Comment:
It is quite confusing that `precision_f32` and `precision_bf16` have very similar naming, but different behavior: the first one always returns non-empty vector, but the second one - only if bf is supported. This difference should be highlighted in the name:
`precision_bf16_if_supported` maybe?
or at least `get_supported_precision_f32` and `get_supported_precision_bf16`?

Reply:
Renamed to `precision_bf16_if_supported` because `fp32` is always supported. Thanks